src/Icon.js

```diff
@@ -70,7 +70,18 @@ const allKeys = Object.keys({
 })
 
 Icon.propTypes = {
-  name: PropTypes.oneOf(allKeys).isRequired,
+  name: props => {
+    const name = props.name
```

Comment:
I think the second argument is the `name`, which means you could skip this line, right?